src/js/components/Select/Select.js

```diff
@@ -284,6 +285,16 @@ const Select = forwardRef(
 
     const iconColor = getIconColor(theme);
 
+    let ariaLabelledBy;
+    if (
+      formContext?.useFormField({})?.inForm &&
+      id &&
+      !ariaLabel &&
+      !placeholder
+    ) {
+      ariaLabelledBy = `grommet-${id}__input__label ${id}`;
```

Comment:
@jcfilben @britt6612 Sorry for the last minute review comment, but was wondering why it's `grommet-${id}__input__label` and not just `grommet-${id}__input__label`?

Reply:
because of the `_input` 
https://v2.grommet.io/formfield#htmlFor